app/src/main/res/values/strings.xml

```diff
@@ -1204,14 +1204,15 @@ Alternatively, you can leave a note (with a photo)."</string>
     <string name="quest_airConditioning_name_type_title">"Does %1$s (%2$s) have air conditioning?"</string>
 
     <string name="no_camera_permission_warning_title">Camera permission</string>
-    <string name="no_camera_permission_warning">The permission is necessary to be able to measure distances with augmented reality.</string>
+    <string name="no_camera_permission_warning">The permission is necessary to be able to measure distances with the camera.</string>
     <string name="ar_measure">Measure</string>
     <string name="ar_core_error_sdk_too_old">This app is too old, try updating.</string>
     <string name="ar_core_tracking_error_bad_state">Something went wrong. Try restarting the measuring.</string>
     <string name="ar_core_tracking_error_insufficient_light">Too dark here</string>
     <string name="ar_core_tracking_error_excessive_motion">Too much movement</string>
     <string name="ar_core_tracking_error_insufficient_features">Insufficient visual features here</string>
     <string name="ar_core_tracking_error_camera_unavailable">Camera is used by another app</string>
+    <string name="default_disabled_msg_no_ar">You need a measure tape to solve this quest because measuring with the camera is not supported on this device.</string>
```

Comment:
According to [Wikipedia], it should be "tape measure" or "measuring tape", but not "measure tape". @smichel17 @peternewman which one sounds better?

Reply:
I **slightly** prefer "measuring tape", but I suspect it is a regional preference. In the line above, I would pick "tape measure" to avoid using the word "measuring" later in the sentence.

I also think it would be best to make the second part active voice ("this device does not support measuring with the camera"). However, it's a pet peeve of mine when apps tell me what "my device" cannot do when it's actually a software limitation (Google Play Services not installed), so I would prefer a phrasing like this:

```suggestion
    <string name="default_disabled_msg_no_ar">You need a measuring tape to solve this quest because StreetComplete is unable to measure with the camera on this device.</string>
```

Or a more awkward but more technically accurate version:

```suggestion
    <string name="default_disabled_msg_no_ar">You need a tape measure to solve this quest because this device seems not to support measuring with the camera on this device.</string>
```